modules/utility/init.zsh

```diff
@@ -175,9 +175,13 @@ elif (( $+commands[wget] )); then
   alias get='wget --continue --progress=bar --timestamping'
 fi
 
-# Resource Usage
-if (( $+commands[pydf] )); then
-  alias df=pydf
+if zstyle -s ':prezto:module:utility' pydf 'yes'; then
```

Comment:
Alternatively we could use `if zstyle -t ':prezto:module:utility:df' color` which lines up with how it's handled in `diff`